shared/common-adapters/user-proofs.desktop.js

```diff
@@ -24,7 +24,7 @@ function MissingProofRow ({missingProof}: {missingProof: MissingProof}): React$E
           </Text>
         </Box>
       </Box>
-      <Icon type={'iconfont-proof-placeholder'} style={{...styleStatusIcon, color: missingColor}} />
+      <Icon type={'iconfont-proof-placeholder'} style={{...styleStatusIcon, color: globalColors.black_10}} />
```

Comment:
No, for some reason the placeholder certificate icon must be black_10 for it too not to look darker than the other items on the row, see design: https://zpl.io/xjWXi